pages/en/zkapps/how-to-test-a-zkapp.mdx

```diff
@@ -37,8 +37,97 @@ Because we are using Jest, it's helpful to break down all functionality of your
 
 For examples of existing tests, we recommend creating a template example using the Mina zkApp CLI via `zk project <name>` and examining the test file there. [You will see a basic example of a few tests](https://github.com/o1-labs/zkapp-cli/blob/main/templates/project-ts/src/Add.test.ts) that deploy and update the state on a smart contract using Jest.
 
+#### Creating a local blockchain
+
+The `Mina.LocalBlockchain()` method specifies a mock Mina ledger of accounts that runs locally. It contains logic for updating the ledger that can be used to test your smart contract locally.
```

Comment:
I read your commits out of context and was confused at first about what this is about. Would it make more sense to intro this with the use case / purpose for why someone would need to create a local blockchain? Are there an other use cases other than testing zkApps? 

Ex. "One way of testing your smart contract is to run it locally on a mock blockchain using built-in methods in the Mina zkApp CLI."